### PR TITLE
Clean up BuildFiles under PhysicsTools/

### PR DESCRIPTION
**Pull request by fwyzard**
Clean up BuildFile.xml filea with `xmllint --format`.

### CODE REVIEW

**Comment by fwyzard** [comment]
@cmsbuild, please test

**Comment by fwyzard** [comment]
@cmsbuild, please test

**Comment by peruzzim** [comment]
+xpog

**Comment by perrotta** [comment]
+1
- Technical
- Changes are only in white spaces and empty lines
- Jenkins tests pass
- (Trying to clean up the queue of pendig pull requests for reconstruction)
- "DQMHistoSizes: changed ( 10024.0,... ): 21.549 KiB ParticleFlow/PackedCandidates" due to testing this together with #29628

File: PhysicsTools/FWLite/bin/BuildFile.xml [hunk 1]
```diff
@@ -1,13 +1,20 @@
-<use   name="root"/>
```
Comment (silviodonato) [inline PhysicsTools/FWLite/bin/BuildFile.xml:1]: @fwyzard

Comment (fwyzard) [reply PhysicsTools/FWLite/bin/BuildFile.xml:1]: sorted

File: PhysicsTools/PatExamples/bin/BuildFile.xml [hunk 2]
```diff
@@ -1,18 +1,33 @@
-<use   name="root"/>
-<use   name="FWCore/FWLite"/>
```
Comment (fwyzard) [inline PhysicsTools/PatExamples/bin/BuildFile.xml:2]: sorted

File: PhysicsTools/PythonAnalysis/test/BuildFile.xml [hunk 3]
```diff
@@ -1,28 +1,22 @@
-#To make sure we rebuild this package when python_tools package updated
```
Comment (fwyzard) [inline PhysicsTools/PythonAnalysis/test/BuildFile.xml:1]: @silviodonato this comment got removed, do you want me to add it back ?

File: PhysicsTools/SelectorUtils/bin/BuildFile.xml [hunk 4]
```diff
@@ -1,13 +1,15 @@
-<use   name="DataFormats/FWLite"/>
-<use   name="FWCore/FWLite"/>
-<use   name="FWCore/ParameterSet"/>
-<use   name="FWCore/ParameterSetReader"/>
-<use   name="root"/>
```
Comment (fwyzard) [inline PhysicsTools/SelectorUtils/bin/BuildFile.xml:5]: sorted

File: PhysicsTools/UtilAlgos/bin/BuildFile.xml [hunk 5]
```diff
@@ -1,9 +1,11 @@
-<use   name="root"/>
-<use   name="boost_program_options"/>
-<use   name="PhysicsTools/UtilAlgos"/>
-<use   name="FWCore/ParameterSetReader"/>
-
+<use name="boost_program_options"/>
+<use name="root"/>
```
Comment (fwyzard) [inline PhysicsTools/UtilAlgos/bin/BuildFile.xml:2]: sorted

File: PhysicsTools/PythonAnalysis/test/BuildFile.xml [hunk 6]
```diff
@@ -1,28 +1,22 @@
...
 <use name="python_tools"/>
```
Comment (fwyzard) [inline PhysicsTools/PythonAnalysis/test/BuildFile.xml:1]:
```suggestion
<!-- To make sure we rebuild this package when python_tools package updated -->
<use name="python_tools"/>
```

**Comment by fwyzard** [comment]
@cmsbuild, please test

**Comment by perrotta** [comment]
+1
- Same as https://github.com/cms-sw/cmssw/pull/29692#issuecomment-624521097

**Comment by peruzzim** [comment]
+xpog

**Comment by santocch** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**